pkg/util/config/feature_gate.go

```diff
@@ -37,6 +37,8 @@ const (
 	// values override the default set by allAlpha.
 	// e.g. allAlpha=false,newFeature=true will result in newFeature=true
 	allAlpha = "allAlpha"
+
+	externalTrafficLocalOnly = "externalTrafficLocalOnly"
```

Comment:
Should check AllAlpha gate as default. If the feature-specific flag is provided, it overrides.